src/pybind/mgr/dashboard/frontend/src/app/shared/api/multi-cluster.service.ts

```diff
@@ -169,4 +175,18 @@ export class MultiClusterService {
 
     return this.http.get<object>('api/multi-cluster/check_token_status', { params });
   }
+
+  showPrometheusDelayMessage(showDeletionMessage?: boolean) {
+    if (showDeletionMessage !== undefined) {
```

Comment:
if you give the `showDeletionMessage` true or false initially, then you don't need to check whether it defined or undefined. And I don't really understand the purpose of this function. You are just returning whatever value which is passed to the function.

Reply:
We need this message for inter communication between the overview component and the cluster list component. If a user disconnects a cluster from the cluster-list page, we should get a flag in the overview page to show a message that disconnected cluster's data shall be removed after ~ 5 minutes